static/js/file_select.js

```diff
@@ -44,59 +46,36 @@ function ValidateURL(fileSelectURL) {
     return Regex.test(fileSelectURL);
 }
 
-//Format object for table display
-function formatForTable(entry) {
-    //console.log("entry", entry.type,entry);
-    for( key in entry) {		
-		var allmin = []
-		if ( key == "minerals" )	{
-			for ( k in entry[key] )	{
-				allmin.push(entry[key][k]["id"])
-			}
-			entry[key] = allmin;
-		}
-		else if ( key == "location_coords" )	{
-			var point= entry[key].substr(entry[key].indexOf('('), (entry[key]).length);
-			//console.log(point)
-			entry[key] = point;
-		}
-		else if ( key == "collection_date" )	{
-			var d = new Date(entry[key]);
-			entry[key] = d.toDateString();
-		}
-		else {
-			entry[key] = entry[key];
-		}
-    }
-    return entry
-}
-
 //Submit the URL
 function ParseFileForUpload() {
     document.getElementById('msgbanner').innerHTML = "";
-    var Checked = null;
+    template = null;
     if (document.getElementById('Samples').checked) {
-        Checked = "SampleTemplate";
+        template = "SampleTemplate";
     }
     else if (document.getElementById('ChemicalAnalyses').checked) {
-        Checked = "ChemicalAnalysesTemplate";
+        template = "ChemicalAnalysesTemplate";
     }
     else if (document.getElementById('Images').checked) {
-        Checked = "Images";
+        template = "Images";
     }
-    if (fileSelectURL == null || Checked == null) {
+    if (fileSelectURL == null || template == null) {
         return;
     }
     //Send Checked and URL as JSON using POST
     var data = {};
     data["url"] = fileSelectURL;
-    data["template"] = Checked;
+    data["template"] = template;
+    sendTopython(data);
+}
+
+function sendTopython(data) {
```

Comment:
Might want to consider renaming this function and adding a one-liner comment at least explaining what it does